### DISCUSSION TITLE
Support macOS Version 10.12 +

### DISCUSSION THREAD

**Discussion by maxhuckerby** [sipsorcery-org/sipsorcery#777] [Q&A]
Hi,

I have a Xamarin.Mac application with a WebRTC library but I am looking to change over. I see you have support for OSX but does this include the latest macOS versions? Is it the full library available for Xamarin.Mac or are there some areas where you haven't added the mac support?

**Comment by sipsorcery** [comment]
I haven't done any tests on oxs for the last 18 months or so. There's no reason it shouldn't work and the osx automated tests are still passing.
